src/sentry/notifications/notifications/base.py

```diff
@@ -30,22 +30,6 @@ class BaseNotification(abc.ABC):
     def __init__(self, organization: Organization):
         self.organization = organization
 
-    @property
-    def org_slug(self) -> str:
```

Comment:
@mgaeta why remove these methods to access commonly used fields?

Reply:
We're updating the documentation on `BaseNotification` and we're not going to be keeping every method and property. Using `self.organization.slug` is more explicit and simpler.

Reply:
@mgaeta I understand the intent behind adding documentation, but I believe that simple property functions like `org_slug` are pretty obvious in what they do (and if not, it's easy to add a short sentence to a document). And these properties add an abstraction layer between the interface and the underlying data model which means you can refactor the latter without updating the interface. And most importantly, it just takes fewer characters to write `self.org_slug` instead of ` self.organization.slug`.

Reply:
This abstraction is not needed since `organization` is defined on `BaseNotification`. That it is fewer characters is not a legitimate consideration. Finally, that this is being used in getsentry means that any changes that we'd want to do are slowed down by having to update two repositories.

Reply:
ok, it's fine to remove these shorthand properties